Source/BoundaryConditions/PML.cpp

```diff
@@ -1105,6 +1105,41 @@ PML::CopyToPML (MultiFab& pml, MultiFab& reg, const Geometry& geom)
   WarpXCommUtil::ParallelCopy(pml, reg, 0, 0, 1, IntVect(0), ngp, period);
 }
 
+amrex::DistributionMapping PML::MakeSimilarDM (const amrex::BoxArray& ba,
+                                               const amrex::MultiFab& mf,
+                                               const amrex::IntVect& ng)
+{
+    amrex::Vector<int> pmap(ba.size());
+    const amrex::DistributionMapping& mf_dm = mf.DistributionMap();
+    const amrex::BoxArray& mf_ba = mf.boxArray();
```

Comment:
```suggestion
    const amrex::BoxArray& mf_ba = amrex::convert(mf.boxArray(),ba.ixType());
```
In PML case, `ba` is cell centered, but `mf` is unlikely to be cell centered too.  Alternatively we could also take BoxArray and DistributionMapping instead of mf.